drivers/periph_common/Kconfig.wdt

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+menuconfig MOD_PERIPH_WDT
+    bool "Watchdog Timer peripheral driver"
+    depends on HAS_PERIPH_WDT
+    select MOD_PERIPH_COMMON
+
+if MOD_PERIPH_WDT
+
+if TEST_KCONFIG
```

Comment:
`MOD_PERIPH_WDT` is also a symbol added during migration, it should depend on `TEST_KCONFIG`